cypress/fixtures/material/instant-loan/holdings.json

```diff
@@ -24,7 +24,7 @@
             "available": false,
             "periodical": null,
             "materialGroup": {
-              "name": "standard",
+              "name": "31 dages lånetid til alm lånere",
```

Comment:
Consider updating the test instead of the test data.

"31 dages lånetid til alm lånere" is not really a value which can be expected for material group names. Such alterations to fixtures which have been copy/pasted from live data make the fixture difficult to maintain.

If a change is necessary and if you want to do something else then update the test to react to the name "standard" instead.